docs/resources/os.md.erb

```diff
@@ -138,3 +138,24 @@ For example, both of the following tests should have the same result:
         its('processes') { should include 'xinetd' }
       end
     end
+
+## os.windows? helpers
+
+Use `os.windows*` helpers to enable more granular testing of Windows versions.
```

Comment:
It does not feel right to me to attach all those helpers to the os resource. It scope should be very limited.

Reply:
Would it make sense to introduce a `windows` resource? I am not sure. What do you think @adamleff

Reply:
I don't think that feels right, either. Are we then going to need to have a `centos` and `linux` resource also?

Reply:
Here is an example of the windows helpers in action.
https://github.com/mattray/inspec-profile-wannacry-exploit/blob/use_windows_helpers/controls/hotfixes.rb#L60

There are lots of security controls that are only applicable to certain versions of Windows. The intention is to provide a user-friendly means to know which version of Windows you're working with without having to parse the name or know the arcane versioning.